lib/grape/config.rb

```diff
@@ -5,6 +5,7 @@ module Config
     class Configuration
       ATTRIBUTES = %i[
         param_builder
+        on_unknown_validator
```

Comment:
Maybe `on_load_validator`?

Reply:
I like `on` prefix as it points on some "callback" implementation. maybe `on_validator_not_found` is even better.
though I have no strong opinion of this, if `load_validator` will documented properly the it's fine)

Reply:
I was looking at `param_builder` as a reference, would just `validator` work and be more future-proof?